Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: help \
+
+.DEFAULT_GOAL := help
+
+
+help: ## Display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	coverage erase
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+diff_cover: test ## find diff lines that need test coverage
+	diff-cover coverage.xml
```

Comment:
Also add this line to the end of test command so that whenever someone runs `make test` they get the diff coverage summary as well.